### PR TITLE
Added: initial apple tv support

### PR DESCRIPTION
**Pull request by equinox2k**
Added update to compile libs for Apple TV, please review as although appears to work the lipo phase complains some architecture already exists. Ive done in hope will save time for official AppleTV support.

### CODE REVIEW

File: contrib/bootstrap [hunk 1]
```diff
@@ -303,13 +350,19 @@ OS="${HOST#*-}" # strip architecture
 echo "The host is $OS"
 case "${OS}" in
 	apple-darwin*)
-		if test -z "$BUILDFORIOS"
+		if test -z "{$BUILDFORIOS}{$BUILDFORAPPLETV}"
 		then
-			check_macosx_sdk
-			add_make_enabled "HAVE_MACOSX" "HAVE_DARWIN_OS" "HAVE_BSD"
+      check_macosx_sdk
```
Comment (zilongshanren) [inline contrib/bootstrap:355]: these indentations are wrong here.

**Comment by zilongshanren** [comment]
@equinox2k  
:+1:  Thanks for adding AppleTV support.

**Comment by zilongshanren** [comment]
@equinox2k  Does your patch work for all the dependencies of cocos2d-x?

**Comment by ricardoquesada** [comment]
yeah, appletv support is great.

**Comment by pyrosphere** [comment]
@elvman already has all libs building for tvOS:
https://github.com/elvman/cocos2d-x-3rd-party-libs-src/tree/tvos

**Comment by zilongshanren** [comment]
@equinox2k 

Since @elvman  has sent a full patch to support AppleTV.
So I will close this PR and let's move to that one.

https://github.com/cocos2d/cocos2d-x-3rd-party-libs-src/pull/67

**Pull request closed without merging**